docs/user_manual/requisitions.md

```diff
@@ -0,0 +1,112 @@
+
+On the Create Requisition page, you can give the requisition an alias and choose an assay. The alias may refer to a
+requisition form stored in a separate system, or may simply be a friendly identifier for the requisition. You must save
+the requisition using the "Save" button at the top right before you can add samples. See below for information about
+adding samples and other tasks you can complete from the Edit Requisition page.
+
+### Editing Requisitions
+
+After saving a new requisition from the Create Requisition page, you will be taken to the Edit Requisition page. To get
+to the Edit Requisition page for existing requisitions, you can click the requisition alias on the Requisitions list
+page, or in the Requisition field of the Edit Sample page.
+
+The Assay Information section at the top of the page allows you to modify the requisition's alias, assay, and stopped
+status. Be sure to click the "Save" button at the top right to apply your changes.
+
+You can add or remove samples from the requisition using the Samples table below. To add samples, click the "Add"
+button in the toolbar. Enter sample names, aliases, or barcodes in the dialog and click Search to find and add the
+samples. To remove samples, select them in the list and click the "Remove" button in the toolbar. The Samples list may
+also be used to perform other actions on the samples.
+
+Farther down the page, there are more lists showing all of the derived samples (if you are using detailed sample mode),
+libraries, library aliquots, and runs. At the very bottom of the page is the requisition's change log.
+
+### Attaching Files
+
+You can attach any type and number of files to a requisition. This feature might be used to attach QC output, reports,
+or any other relevant case files. For more information, see the [Attachments section](../attachments/).
+
+### Attaching QCs
+
+Any number of quality control measures may be recorded for a requisition. For more information on this feature, see the
+[QCs section](../qcs/).
+
+## Assays
+
+An assay details the work intended to be performed for a requisition, and quality control requirements for that work.
+The requirements are defined using [metrics](../type_data#metrics).
+
+### Assays List
+
+To get to the Assays list page, click to expand the Configuration menu within the main navigation menu, then click the
+"Assays" link within. All of the existing assays are shown here, and the toolbar at the top of the list contains
+controls for working with assays.
+
+### Creating Assays
+
+To create a new assay, you must be a MISO administrator. Click the "Add" button in the toolbar at the top of the Assays
+list. This takes you to the Create Assay page. In the top section of the page, you can enter the basic information for
+the assay, including its alias, version, and description.
+
+Below, controls in the Metrics table allow you to define the assay's metrics. To add a metric, click the "Add" button.
```

Comment:
I've clicked the "Add" button and found that the "Add Metric" popup is entirely blank.

Reply:
You have to create the metrics first before you can add them to an assay. I'll add a message to make that more obvious when there are no metrics though

Reply:
Ah, that makes sense :+1: